test/test_patches.py

```diff
@@ -158,19 +158,19 @@
 -    return x + y;
 +    return (x + y);
  }
- 
```

Comment:
These lines need to be left as-is in test_patches.py.  It is a patch-compatible formatted diff that works across Linux and various BSD systems.